### DISCUSSION TITLE
Update embeddebles?

### DISCUSSION THREAD

**Discussion by rubiin** [mikro-orm/mikro-orm#3545] [Q&A]
How do we update embeddables?
The entity has an array of embeddables and i am trying to update one of them with new values

**Comment by rubiin** [comment]
@B4nan

**Reply by B4nan** [reply]
I would appreciate if you dont spam multiple channels. If you do so, dont tag me in multiple places.

As said on slack, just modify the state and flush. And next time provide at least code snippets of what you are trying to do, instead of asking broad questions :]
